firefox_puppeteer/ui/update_wizard/dialog.py

```diff
@@ -0,0 +1,126 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime
+
+from marionette_driver import By, Wait
+
+from ..windows import BaseWindow
+from ...api.prefs import Preferences
+from ...api.software_update import SoftwareUpdate
+from .wizard import Wizard
+
+
+# TODO: Bug 1143020 - Subclass from BaseDialog ui class with possible wizard mixin
+class UpdateWizardDialog(BaseWindow):
+    """Representation of the old Software Update Wizard Dialog."""
+    window_type = 'Update:Wizard'
+
+    dtds = [
+        'chrome://branding/locale/brand.dtd',
+        'chrome://mozapps/locale/update/updates.dtd',
+    ]
+
+    properties = [
+        'chrome://branding/locale/brand.properties',
+        'chrome://mozapps/locale/update/updates.properties',
+    ]
+
+    # For the old update wizard, the errors are displayed inside the dialog. For the
+    # handling of updates in the about window the errors are displayed in new dialogs.
+    # When the old wizard is open we have to set the preference, so the errors will be
+    # shown as expected, otherwise we would have unhandled modal dialogs when errors are raised.
+    # See: http://mzl.la/1GUoNMU or http://mzl.la/1BLrBsC
```

Comment:
Those will give us nasty very long lines which are hard to read. What's the reason for your request?